doc/conf.py

```diff
@@ -23,16 +26,13 @@
 # Built-in Sphinx extensions are already contained in the imported variable
 # here we add external extensions, which must also be added to requirements.txt
 # so that RTD can import and use them
-extensions.extend(["numpydoc"])
+extensions.extend(["numpydoc", "ext.costfunctionslist"])
 
 # mock import for autodoc
-autodoc_mock_imports = ["numpy", "mpi4py", "petsc4py", "baseclasses", "adflow.om_adflow"]
+autodoc_mock_imports = ["numpy", "mpi4py", "petsc4py", "baseclasses"]
 
 # bibtex sources
 bibtex_bibfiles.extend(["citations.bib"])
 
-# intersphinx
-intersphinx_mapping = {
-    "mach-aero": ("https://mdolab-mach-aero.readthedocs-hosted.com/en/latest", None),
```

Comment:
`intersphinx_mapping` is defined in sphinx-mdolab-theme, so this is unnecessary.

As for the Sphinx extension, enough of the extension is specific to pyADflow that it will be difficult to generalize nicely.